servatrice/src/servatrice_database_interface.cpp

```diff
@@ -1128,3 +1130,91 @@ int Servatrice_DatabaseInterface::checkNumberOfUserAccounts(const QString &email
 
     return 0;
 }
+
+bool Servatrice_DatabaseInterface::addForgotPassword(const QString &user)
```

Comment:
See comment : https://github.com/Cockatrice/Cockatrice/pull/2393#discussion_r100698083
Since everyone seemed really interested in a table for auditing , the very next PR I intend to submit (but left it out of this PR for clarity) is to create an audit table that will start with auditing information on registration, forgot password, and log queries done by admins that I can see the possibility of expanding to other aspects at some point.  Have to start some were though.